predict_with_corr.py

```diff
@@ -0,0 +1,68 @@
+from statistics import NormalDist
+import numpy as np
+
+# From https://www.reddit.com/r/berkeley/comments/kbvxd2/calculating_your_grade_in_cs70_and_other_curved/
+
```

Comment:
add in: 
def mt1_zscore(raw_score):
    # Updated 4/5/22
    mean = 132.72
    std = 39.01
    return (raw_score - mean)/std

Reply:
Added